data/moves.js

```diff
@@ -10294,8 +10278,7 @@ let BattleMovedex = {
 				let move = exports.BattleMovedex[i];
 				if (i !== move.id) continue;
 				if (move.isZ || move.isNonstandard) continue;
-				// @ts-ignore
-				if (effect.noMetronome.includes(move.id)) continue;
+				if (effect.noMetronome && effect.noMetronome.includes(move.id)) continue;
```

Comment:
Also, your checking instead of `ts-ignore` I'm generally against. `ts-ignore` is correct, this should be an assertion, not a check.